src/main.ts

```diff
@@ -1,4 +1,6 @@
 import './assets/main.css'
+import './style.css'
```

Comment:
Why not move import the tailwind stuff in `main.css`? (instead of creating a whole other css file)